propagator/toolbox.py

```diff
@@ -385,14 +384,22 @@ def updateParameters(self, parameters):
         params = self._get_parameter_dict(parameters)
         param_vals = self._get_parameter_values(parameters)
         ws = param_vals.get('workspace', '.')
+        vc = params['value_columns']
 
         with utils.WorkSpace(ws):
+            ml = param_vals['monitoring_locations']
             if params['ml_type_col'].altered:
-                ml = param_vals['monitoring_locations']
                 col = param_vals['ml_type_col']
                 values = utils.unique_field_values(ml, col).tolist()
                 params['included_ml_types'].filter.list = values
 
+            if params['monitoring_locations'].value:
+                fields = analysis._get_wq_fields(ml, ['dry', 'wet'])
+                self._set_filter_list(vc.filters[0], fields)
+                self._set_filter_list(vc.filters[1], list(analysis.AGG_METHOD_DICT.keys()))
+
+            self._update_value_table_with_default(vc, 'average')
```

Comment:
@insect1285 you might want to look at the changes to see how to make the value table a dropdown menu

The `_set_filter_list` and `_update_value...` methods are in the mixin class defined in base_tbx.py